### ISSUE TITLE
resetClass/es behavior

### ISSUE THREAD

**Issue by chasegiunta** [verbb/formie#1113]
### Describe the bug

_This may not be a bug, and if so, please close! Potentially could use a documentation clarification._

`[field-handle].resetClass` removes all `fui` related classes, which is great, and retains custom classes declared via the field's advanced tab,

`themeConfig.resetClasses` removes _all classes_, even custom ones defined in the field's advanced tab.

I _kind of_ expected the themeConfig.resetClasses option to retain custom defined classes in fields settings, but I could also see cases where the full reset of everything is necessary.

### Steps to reproduce

1. Observe `field.resetClass` behavior
2. Observe `themeConfig.resetClasses` behavior

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

4.2.5.2

### Plugin version

2.0.12

**Comment by engram-design** [comment]
Hmmm, yes that's a good point! It should only remove the default `fui-*` ones. But this might be tricky in practice as removing the classes is done after merging the defaults with config-specified ones, and event-specified ones.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.12",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.13](https://github.com/verbb/formie/releases/tag/2.0.13)
